.circleci/fetch_merge_commit.sh

```diff
@@ -25,17 +25,17 @@ err=0
 GREEN() { echo -e "\n\033[0;32m$1\033[0m"; }
 RED() { echo -e "\n\033[0;31m$1\033[0m"; }
 
-# CIRCLE_PULL_REQUEST is present for PR builds, and absent for push builds.
-if [[ -z "$CIRCLE_PULL_REQUEST" ]]; then
+# CIRCLE_PR_NUMBER is present for PR builds, and absent for push builds.
+if [[ -z "$CIRCLE_PR_NUMBER" ]]; then
   echo $(GREEN "Nothing to do because this is not a PR build.")
   exit 0
 fi
 
 # Make sure the PR is on ampproject/amphtml and not on a fork.
-if [[ ! "$CIRCLE_PULL_REQUEST" =~ ^https://github.com/ampproject/amphtml* ]]; then
-  echo $(RED "This is a PR build, but on a repo other than ampproject/amphtml.")
-  exit 1
-fi
+# if [[ ! "$CIRCLE_PULL_REQUEST" =~ ^https://github.com/ampproject/amphtml* ]]; then
+  # echo $(RED "This is a PR build, but on a repo other than ampproject/amphtml.")
+  # exit 1
+# fi
```

Comment:
Instead of commenting this out, can we make it a non-blocking warning instead?

```suggestion
if [[ ! "$CIRCLE_PULL_REQUEST" =~ ^https://github.com/ampproject/amphtml* ]]; then
  echo $(YELLOW "This build is linked to a repo other than ampproject/amphtml.")
fi
```